code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -422,6 +422,7 @@
 	pixel_x = -64
 	pixel_y = -64
 	unacidable = 1
+	mouse_opacity = 0
```

Comment:
I AM LITERALLY ONE-LINE FIXING, HERE, AND THIS ONE LINE IS A BUFF THE SIZE OF NEVADA